resources/htmlTemplate/ExceptionsTocHTMLTemplate.html

```diff
@@ -135,14 +135,17 @@ <h1>License Exceptions</h1>
       <p>The SPDX License List includes a list of exceptions. These exceptions grant an exception to a license condition or additional permissions beyond those granted in a license; they are not stand-alone licenses. Exceptions are added to a license using the <a href="https://spdx.org/specifications"><strong>License Expression</strong></a> operator, "WITH".</p>
 
       <ul style="list-style-type: square; padding: 5px;">
-            <li style="margin:5px 0 5px 10px;"><a href="index.html"><strong>SPDX License List</strong></a>: The list of commonly found free and open source licenses for the purposes of being able to easily and efficiently identify such licenses.</li>
-			<li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML"><strong>Master Files</strong></a>:  The HTML pages you see here are generated from the master files for the SPDX License List.  </li>
-            <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-data"><strong>Data Files</strong></a>:  Machine readable files describing all of the licenses and license exceptions.</li>
-            <li style="margin:0px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/license-list-overview"><strong>Overview</strong></a>: General information about the SPDX License List, including principles for inclusion of a license and an explanation of the fields contained on the list.</li>
-            <li style="margin:5px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>Matching Guidelines</strong></a>:  Guidelines for what constitutes a license match to the SPDX License List. For licenses that include markup, the license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
-            <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md"><strong>Request New License</strong></a>: Instructions for proposing a license or exception be added to the SPDX License List.</li>
+            <li style="margin:5px 0 5px 10px;"><a href="index.html"><strong>SPDX License List</strong></a></li>
+            <li style="margin:5px 0 5px 10px;">The <a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>matching guidelines</strong></a> define what constitutes a license or exception match. The license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
```

Comment:
The URL here can be updated to https://spdx.dev/license-list/matching-guidelines/